tests/www/views/test_session.py

```diff
@@ -97,7 +97,7 @@ def test_check_active_user(app, user_client):
     user.active = False
     resp = user_client.get("/home")
     assert resp.status_code == 302
-    assert "/logout" in resp.headers.get("Location")
+    assert "/login/?next=http%3A%2F%2Flocalhost%2Fhome" in resp.headers.get("Location")
```

Comment:
This is result of https://github.com/dpgaspar/Flask-AppBuilder/pull/2192 from FAB. And I think it's a better behaviour, because it will return the user back to the page they were originally in case someone re-activates them after they've been logged out.